### PR TITLE
Refactor sensor defaults

### LINKED ISSUE #196: app_sensor: Refactor sensor defaults from .c to .h
Check BME280 for an example.

### PR DESCRIPTION
**Pull request by niksiboxi**
Refactored sensors' default configuration from *.c to *.h following BME280 configuration pattern. Resolves #196

### CODE REVIEW

File: src/app_sensor.c [hunk 1]
```diff
@@ -1,21 +1,21 @@
-#include "app_config.h"
```
Comment (ojousima) [inline src/app_sensor.c:1]: App config is intentionally the first include, that let's app_config override any defaults

**Review by ojousima — CHANGES_REQUESTED** [review]

File: src/app_sensor.c (hunk 1 above)
Comment (niksiboxi) [reply src/app_sensor.c:1]: Fixed in bde68be

Comment (ojousima) [reply src/app_sensor.c:1]: Easiest way to get the coverage to required level is to add tests for gyroscope data handling. Check how accX, accY, accZ and accXYZ are tested at https://github.com/ruuvi/ruuvi.firmware.c/blob/df0c262db98cbc92b865cd2f5ff5e89b767aa1b9/test/test_app_sensor.c#L761 and repeat the test cases for gyroscope. Definitions are at ruuvi.endpoints.c/ruuvi_endpoints.h

**Review by DG12 — COMMENTED** [review]
At over 1,000 lines maybe this should be broken up??

**Comment by niksiboxi** [comment]
@DG12 Some of the extra lines are due to the use of code styling.

**Comment by ojousima** [comment]
> At over 1,000 lines maybe this should be broken up??
No, test files cannot be broken up due to Ceedling functionality.

**Review by ojousima — APPROVED** [review]
Thanks

**Comment by ojousima** [comment]
Something is missing, please check the Travis build log for details

**Pull request merged**